actions/universal-remove-banners.js

```diff
@@ -24,14 +38,14 @@ module.exports = (course, item, callback) => {
         //taken out of the page.
         var changeBool = false;
 
-        if (images.length < 0) {
+        if (images.length === 0) {
             return;
         } else {
             images.each((index, image) => {
                 var alt = $(image).attr('alt');
 
-                if (alt !== '' && typeof alt !== 'undefined') {
-                    if (alt.match(/course banner/gi)) {
+                if (alt !== '' && alt === 'undefined') {
```

Comment:
This looks like it is going to check to make sure the string is not empty, but must equal a string of 'undefined.' Is this correct?

Reply:
This should be checking to see if it is undefined. I apologize for this mistake. I'll make this fix later today.